schema/handler_test.go

```diff
@@ -0,0 +1,79 @@
+package schema_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"tidbyt.dev/pixlet/runtime"
+)
+
+var handlerSource = `
+load("schema.star", "schema")
+
+def assert(success, message=None):
+    if not success:
+        fail(message or "assertion failed")
+
+def foobar(param):
+    return "derp"
+
+h = schema.Handler(
+    handler = foobar,
+    type = schema.HandlerType.String,
```

Comment:
Would something like this feel better to use?
```
schema.StringHandler
schema.OptionsHandler
```

Reply:
Not sure. I think I did this way to group it under some object to make it clearer that these are "types", whatever that is in this context.
WDYT?